lib/TPP/IR/MatcherUtils.cpp

```diff
@@ -113,6 +113,10 @@ bool isTwoDSubOp(linalg::LinalgOp linalgOp, SmallVectorImpl<Value> *operands) {
   return isTwoDEltWiseOpOfTypeTy<arith::SubFOp>(linalgOp, operands);
 }
 
+bool isTwoDMulOp(linalg::LinalgOp linalgOp, SmallVectorImpl<Value> *operands) {
+  return isTwoDEltWiseOpOfTypeTy<arith::MulFOp>(linalgOp, operands);
```

Comment:
Honestly, these helpers don't really help anything. Can we just move the inner call to the outer users?

Reply:
Fair enough, not important for this PR.